javascript/ql/lib/change-notes/2022-09-08-implicit-read-flowstates.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
+---
+* Fixed an issue in the taint tracking analysis where implicit reads were not allowed by default in sinks or additional taint steps that used flow states.
```

Comment:
JS doesn't use the shared lib, so we can remove this change note.

Reply:
Woops 👍